Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/BoundsControl.cs

```diff
@@ -1229,27 +1361,63 @@ private void ResetVisuals()
                 return;
             }
 
-            links.Reset(active, flattenAxis);
+            
 
             boxDisplay.Reset(active);
             boxDisplay.UpdateFlattenAxis(flattenAxis);
 
             bool isVisible = (active == true && wireframeOnly == false);
-            rotationHandles.Reset(isVisible, flattenAxis);
-            scaleHandles.Reset(isVisible, flattenAxis);
+
+            if (enabledHandles.HasFlag(HandleFlags.Rotation))
+            {
+                rotationHandles.Reset(isVisible, flattenAxis);
+
+                // Links depend on rotation handles for position calculations.
+                links.Reset(active, flattenAxis);
```

Comment:
@thalbern Links cannot be created if rotation handles don't exist. Therefore I'm not sure what it would even mean to reset the links if/when they were never created to begin with?

Reply:
Resolved, using builtin config visibilities instead of flags.